### PR TITLE
Added 7 new Slb resources.

### PR DESCRIPTION
**Pull request by devendra-bhatt-a10**
Added following resources.
health_gateway, health_stat, http2, http_proxy, hw_compress, ssl_cert_revoke and ssl_forward_proxy.
vthunder_slb_imapproxy
vthunder_slb_mysql
vthunder_slb_icap_http
vthunder_slb_l7session
vthunder_slb_mssql
vthunder_slb_mlb
vthunder_slb_l4
vthunder_slb_icap

### CODE REVIEW

File: vthunder/resource_vthunder_slb_health_gateway_test.go [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+	sampling_enable {
+		counters1 = "all"
+	}
+}
+`
+
+//Acceptance test
+func TestAccVthunderHealthGateway_create(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() {
```
Comment (SwapnilKashidA10) [inline vthunder/resource_vthunder_slb_health_gateway_test.go:20]: Is pre Check needed?

File: vthunder/resource_vthunder_slb_health_stat_test.go [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+		counters1 = "all"
+	}
+}
+
+`
+
+//Acceptance test
+func TestAccVthunderHealthStat_create(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() {
```
Comment (SwapnilKashidA10) [inline vthunder/resource_vthunder_slb_health_stat_test.go:21]: Is pre Check needed?

File: vthunder/resource_vthunder_slb_ssl_cert_revoke_test.go [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+var TEST_SLB_SSL_CERT_REVOKE_RESOURCE = `
+resource "vthunder_slb_ssl_cert_revoke" "SSLCertRevoke" {
+	sampling_enable {
+	    counters1 = "all"
+	}
+}
+`
+
+//Acceptance test
+func TestSlbSSLCertRevoke_create(t *testing.T) {
```
Comment (SwapnilKashidA10) [inline vthunder/resource_vthunder_slb_ssl_cert_revoke_test.go:18]: TestACC missing

File: vthunder/resource_vthunder_slb_ssl_forward_proxy_test.go [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+var TEST_SLB_SSL_FORWARD_PROXY_RESOURCE = `
+resource "vthunder_slb_ssl_forward_proxy" "SSLForwardProxy" {
+	sampling_enable {
+	    counters1 = "all"
+	}
+}
+`
+
+//Acceptance test
+func TestSlbSSLForwardProxy_create(t *testing.T) {
```
Comment (SwapnilKashidA10) [inline vthunder/resource_vthunder_slb_ssl_forward_proxy_test.go:18]: TestAcc remaining

File: website/docs/r/vthunder_slb_ssl_cert_revoke.html.markdown [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+layout: "vthunder"
+page_title: "vthunder: vthunder_slb_ssl_cert_revoke"
+sidebar_current: "docs-vthunder-resource-slb-ssl-cert-revoke"
+description: |-
+    Provides details about vthunder SLB ssl cert revoke resource for A10
+---
+
+# vthunder\_slb\_ssl\_cert\_revoke
+
+`vthunder_slb_ssl_forward_proxy` Provides details about vthunder SLB ssl cert revoke
```
Comment (SwapnilKashidA10) [inline website/docs/r/vthunder_slb_ssl_cert_revoke.html.markdown:11]: change to slb_ssl_cert_revoke

File: vthunder/resource_vthunder_slb_health_gateway_test.go (hunk 1 above)
Comment (devendra-bhatt-a10) [reply vthunder/resource_vthunder_slb_health_gateway_test.go:20]: No, it is just an additional check, not compulsary.

**Pull request merged**